### PR TITLE
Update `global` reference

### PR DESCRIPTION
**Pull request by mathiasbynens**

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -404,7 +404,7 @@ Instead, what I'm going to do is list some things that are commonly seen in Java
 
 <sup>[2]</sup> These things are available in both browsers and Node.js, but are non-standard. For Node.js, they are documented/specified by [=Node.js/globals|its documentation=]. For browsers, {{/console}} is specified by the Console Standard [[CONSOLE]], while the rest is specified by the HTML Standard [[HTML]].
 
-<sup>[3]</sup> These are all Node.js-only globals, documented/specified by [=Node.js/globals|its documentation=]. \* <code class=idl>global</code> actually has a chance of becoming a part of ECMAScript and be implemented in browsers as well [[ECMA-262-GLOBAL]].
+<sup>[3]</sup> These are all Node.js-only globals, documented/specified by [=Node.js/globals|its documentation=]. \* Note that unlike <code class=idl>global</code>, <code class=idl>globalThis</code> actually has a chance of becoming a part of ECMAScript and be implemented in browsers as well [[ECMA-262-GLOBAL]].
```
Comment (ExE-Boss) [inline index.bs:407]: `globalThis` is&nbsp;now&nbsp;supported [in&nbsp;all&nbsp;evergreen&nbsp;browsers](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/globalThis#Browser_compatibility).

**Review by ExE-Boss — CHANGES_REQUESTED** [review]

File: index.bs (hunk 1 above)
Comment (sideshowbarker) [reply index.bs]: As long as that comment’s being touched, might also want to change “and be implemented” to “and *being* implemented” — or else change “chance of becoming” to “chance *to become*”

Comment (sideshowbarker) [inline index.bs:407]:
```suggestion
<sup>[3]</sup> These are all Node.js-only globals, documented/specified by [=Node.js/globals|its documentation=]. \* Note that unlike <code class=idl>global</code>, <code class=idl>globalThis</code> actually has a chance of becoming a part of ECMAScript and being implemented in browsers as well [[ECMA-262-GLOBAL]].
```

**Comment by TimothyGu** [comment]
I’ll merge this when I next get to a place where I can regenerate index.html. Haven’t set up Travis CI / GitHub Actions yet!

**Pull request closed without merging**